packages/eslint-config/shared.js

```diff
@@ -151,25 +151,16 @@ module.exports = {
       parser: '@typescript-eslint/parser',
       extends: ['plugin:@typescript-eslint/recommended', 'prettier'],
       rules: {
-        // 'recommended' rules we alter
-        '@typescript-eslint/no-explicit-any': 'warn',
+        // TODO: look into enabling these eventually
+        '@typescript-eslint/no-empty-function': 'off',
+        '@typescript-eslint/prefer-function-type': 'off',
+
+        // Specific 'recommended' rules we alter
         '@typescript-eslint/no-var-requires': 'off',
-        '@typescript-eslint/ban-types': 'warn',
         '@typescript-eslint/no-unused-vars': [
           'error',
           { varsIgnorePattern: '^_', argsIgnorePattern: '^_' },
         ],
-        // Specific 'stylistic' rules we enable
-        '@typescript-eslint/adjacent-overload-signatures': 'error',
-        '@typescript-eslint/no-inferrable-types': 'error',
-        '@typescript-eslint/no-non-null-assertion': 'warn',
-        // Other non-grouped rules we alter
-        'valid-typeof': 'off',
-        camelcase: 'off',
-        '@typescript-eslint/camelcase': 'off',
-        'no-empty-function': 'off',
-        'no-use-before-define': 'off',
-        '@typescript-eslint/no-use-before-define': 'off',
```

Comment:
I suppose this is user-facing - would you rather these changes be reverted? 

I'm not clear on what should be in `eslint-config/shared.js` vs. just used for the repo itself.

Reply:
> I suppose this is user-facing - would you rather these changes be reverted?

For now they can stay. We'll see where we are in our release cycle when v8 becomes stable


> I'm not clear on what should be in eslint-config/shared.js vs. just used for the repo itself.

No one is 🙁  In the future we should look into just totally separating them I think.